api/fixtures/periods.yml

```diff
@@ -2,25 +2,25 @@ App\Entity\Period:
   period1:
     camp: '@camp1'
     description: Hauptlager
-    start: '<(new DateTime((date("Y")+1)."-05-01"))>'
-    end: '<(new DateTime((date("Y")+1)."-05-03"))>'
+    start: '<(new DateTime("2023-05-01"))>'
```

Comment:
war das nötig, um in den tests die daten ausschreiben zu können.
Könntest du damit leben, dass man die response mit den fixtures (eventuell modifiziert) vergleicht?
Es ist schon praktisch, wenn dies automatisch aktualisiert wird, und wir so testdaten haben, die mit floaten.

Wir können die fixtures aber auch einmal im Jahr updaten

Reply:
Ja, ich wollte die Vergleichsdaten in den Tests explizit als String ausschreiben.

Ich hatte es zuerst dynamisch, ähnlich wie hier: 
https://github.com/ecamp/ecamp3/blob/devel/api/tests/Api/Periods/ReadPeriodTest.php#L82

Ist aber nicht so übersichtlich und gerade beim Testen von Timezones ist es schwierig nachzuvollziehen, ob man jetzt gerade das richtige getestet hat oder nicht.

Da wir Test-Daten ja sowieso von Dev-Daten trennen wollen, haben wir so viel Vorteil davon mitfloatende Daten zu haben? Allenfalls wenn wir auf `Camp` oder `Period` DateFilter einbauen, aber auch da können wir die Tests gegenüber einem fixen Datum laufen lassen und müssten nicht mit `today` filtern.

Falls dennoch gewünscht, dann würde ich's in den Tests am ehesten so implementieren:
```
'start' => (date("Y")+1).'-05-01T00:00:00+00:00'
```

Reply:
Ist ok, wir müssen uns einfach im klaren sein, dass wir das warten müssen, wenn wir für die tests aktuelle daten wollen